### DISCUSSION TITLE
Could you animate a whole feature film on an iPad?

### DISCUSSION THREAD

**Discussion by Cjjuntti** [keiwando/toonsquid#616] [General]
So, I’ve had my iPad for about a year and a half, and I’ve used toonsquid for a majority of that time. But now, I’ve recently got davinci resolve for iPad, and I recently watched on YouTube that davinci resolve CAN do 2d animation and you can composite in there too. I tried it on my desktop, and I think it looks great. But that makes me wonder: with toonsquid and davinci resolve fully bought on my iPad, could I make an entire feature film with high production values using JUST my iPad? Let me know if you think it’s possible.

**Comment by Cramos821** [comment]
It’s definitely possible. ToonSquid is very capable. And using davinci resolve to edit the animated clips together, you can make a movie. You can even use the free software garage band to record audio and make music for your movie
